obsplus/events/schema.py

```diff
@@ -0,0 +1,612 @@
+"""
```

Comment:
I'm going to assume this is all there given the tests. I haven't checked through this fully though!

Reply:
Yes, there is at least one test event (#4) which uses every kind of attribute supported by ObsPy. There is a little edge case where the `CompositeTime`'s uncertainties corresponding to int fields don't work (as mentioned above). This is really more of a bug in ObsPy's parsing/typing though and I don't think that it will really effect anyone.